widget/index.js

```diff
@@ -30,6 +30,13 @@ function cropOption(mode, width, height) {
   }
 }
 
+function sanitizeFileName(fileName) {
+  var regexp = /[^A-Za-z0-9_]+/g
+  var extension = fileName.split('.').pop()
+  var name = fileName.substring(0, fileName.length - extension.length)
+  return name.replace(regexp, '') + '.' + extension.replace(regexp, '')
+}
```

Comment:
if a name will contain non-english words only, then the sanitized name will be empty, for example

```
sanitizeFileName('Снимок экрана.png')
```

result - `.png`

Our upload servers set name `noroot` in that cases

Reply:
Fixed